tests_integration/__tests__/__snapshots__/require-pragma.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`requiring a pragma of @format formatting files with the pragma 1`] = `
+"/**
+ * @flow
+ * @format
+ */
+
```

Comment:
hmm.. the whole thing *does* work. not sure what happened here...

Reply:
What do you mean? Isn't it supposed to be working?

By the way, it doesn't have to be an integration test. It can be written as a normal test. You can create a new folder and tweak the config in jsfmt.spec.js